frontend-react/src/pages/submissions/SubmissionDetails.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { MatcherFunction, screen } from "@testing-library/react";
+import ActionDetailsResource from "../../resources/ActionDetailsResource";
+import { success } from "../../resources/fixtures/ActionDetailFixtures";
+import { renderWithRouter } from "../../utils/CustomRenderUtils";
+
+import SubmissionDetails, {
+    DestinationItem,
+    DetailItem,
+} from "./SubmissionDetails";
+
+/*
+    Using the included regex can end up pulling various elements where the
+    value has the parsed timestamp. Use a function
+*/
+const timeRegex: RegExp = /[0-9]{1,2}:[0-9]{1,2} [A,P]M/;
```

Comment:
sonarcloud is also complaining about using `\d` instead of `[0-9]` in both places of the regex

Reply:
@TomNUSDS had fixed an issue that using `\d` caused. It solved the smell but created some crazy kinda security error for SonarCloud. 😬 It was a bothersome event.

Reply:
What timestamp format is typically expected?

Reply:
@meckila This is meant to parse a UTC timestamp `1995-03-15T17:30:00.00000Z` into the strings displayed as "15 Mar 1995" and "5:30 PM".